packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -549,6 +548,13 @@ function accumulateTwoPhaseDispatchesSingle(event) {
 
 const Simulate = {};
 
+const directDispatchEventTypes = new Set([
```

Comment:
Not sure this actually makes sense since those should have a special enter-leave traversal? But this is what the code is already doing.